packs/pathfinder-monster-core/seraptis.json

```
@@ -1147,7 +1147,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p><strong>Requirements</strong> The seraptis has at least one creature @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] or @UUID[Compendium.pf2e.conditionitems.Item.Restrained]</p>\n<hr />\n<p><strong>Effect</strong> s The seraptis's arm mouths gnaw on those creatures, dealing each of them @Damage[(2d6+8)[piercing]] damage with a @Check[type:fortitude|dc:37|basic:true] save. Creatures that fail the save also take @Damage[2d6[bleed]]{2d6 persistent bleed damage}.</p>"
+                    "value": "<p><strong>Requirements</strong> The seraptis has at least one creature @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] or @UUID[Compendium.pf2e.conditionitems.Item.Restrained]</p>\n<hr />\n<p><strong>Effects</strong> The seraptis's arm mouths gnaw on those creatures, dealing each of them @Damage[(2d6+8)[piercing]] damage with a @Check[type:fortitude|dc:37|basic:true] save. Creatures that fail the save also take @Damage[2d6[bleed]]{2d6 persistent bleed damage}.</p>"
```

```suggestion
                    "value": "<p><strong>Requirements</strong> The seraptis has at least one creature @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] or @UUID[Compendium.pf2e.conditionitems.Item.Restrained]</p>\n<hr />\n<p><strong>Effect</strong> The seraptis's arm mouths gnaw on those creatures, dealing each of them @Damage[(2d6+8)[piercing]] damage with a @Check[type:fortitude|dc:37|basic:true] save. Creatures that fail the save also take @Damage[2d6[bleed]]{2d6 persistent bleed damage}.</p>"
```
We can normalize even when Paizo makes a mistake in their formatting